src/main/java/net/minecraftforge/client/event/ContainerScreenEvent.java

```diff
@@ -26,17 +26,17 @@
 /**
  * Event class for handling GuiContainer specific events.
```

Comment:
Missing name change

Reply:
While I will fix this and the other review issues to ensure the javadocs are consistent with the class names, I will state that I deliberately did not touch the javadocs _(excepting those that were touched by removing qualifications and the like)_ because a future PR of mine will completely rewrite those javadocs, making it (at least, to me) a pointless endeavour. 

_(In fact, I have already rewritten the javadocs for this event, see the [event_docs branch on my fork][event_docs_branch])_

[event_docs_branch]: https://github.com/sciwhiz12/MinecraftForge/tree/1.16/event_docs

Reply:
Oh, I thought you wanted to do it all in one go.